From 459ade071534de37f29bb1eedbacdb7b59f05f82 Mon Sep 17 00:00:00 2001
From: Emir SARI <emir_sari@icloud.com>
Date: Tue, 20 Feb 2024 18:50:28 +0000
Subject: [PATCH] Revert "Fix i18n domain"

This reverts commit 8aebe3f54874af2b6ca61cbd1a1fd3fabca00c71. The fix for bug [#480424](https://bugs.kde.org/show_bug.cgi?id=480424) was later fixed in [Plasma Workspace](https://invent.kde.org/plasma/plasma-workspace/-/commit/2eab616a5340af6c36519bc4a8bb51843e9f93d6), and dealt with it. This one broke existing translations. Needs to be cherry-picked into 6.0 (sorry about that).

(cherry picked from commit 84900edcea17bb2070606f3a1520f57aff1d1476)
---
 containments/panel/contents/ui/ConfigOverlay.qml | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/containments/panel/contents/ui/ConfigOverlay.qml b/containments/panel/contents/ui/ConfigOverlay.qml
index 698491e4a1..23e6e2633c 100644
--- a/containments/panel/contents/ui/ConfigOverlay.qml
+++ b/containments/panel/contents/ui/ConfigOverlay.qml
@@ -274,7 +274,7 @@ MouseArea {
                     visible: tooltip.location !== PlasmaCore.Types.TopEdge
                              && (configurationArea.currentApplet?.applet.plasmoid.internalAction("remove")?.enabled ?? false)
                     icon.name: "delete"
-                    text: i18nd("plasma_shell_org.kde.plasma.desktop", "Remove")
+                    text: i18n("Remove")
                     onClicked: {
                         configurationArea.currentApplet.applet.plasmoid.internalAction("remove").trigger();
                         configurationArea.currentApplet = null;
@@ -284,7 +284,7 @@ MouseArea {
                     id: configureButton
                     Layout.fillWidth: true
                     icon.name: "configure"
-                    text: i18nd("plasma_shell_org.kde.plasma.desktop", "Configure…")
+                    text: i18n("Configure…")
                     visible: configurationArea.currentApplet.applet.plasmoid.hasConfigurationInterface
                     onClicked: {
                         configurationArea.currentApplet.applet.plasmoid.internalAction("configure").trigger();
@@ -295,7 +295,7 @@ MouseArea {
                     id: alternativesButton
                     Layout.fillWidth: true
                     icon.name: "widget-alternatives"
-                    text: i18nd("plasma_shell_org.kde.plasma.desktop", "Show Alternatives…")
+                    text: i18n("Show Alternatives…")
                     onClicked: {
                         configurationArea.currentApplet.applet.plasmoid.internalAction("alternatives").trigger();
                         configurationArea.currentApplet = null;
@@ -308,7 +308,7 @@ MouseArea {
                     visible: tooltip.location === PlasmaCore.Types.TopEdge
                              && (configurationArea.currentApplet?.applet.plasmoid.internalAction("remove")?.enabled ?? false)
                     icon.name: "delete"
-                    text: i18nd("plasma_shell_org.kde.plasma.desktop", "Remove")
+                    text: i18n("Remove")
                     onClicked: {
                         configurationArea.currentApplet.applet.plasmoid.internalAction("remove").trigger();
                         configurationArea.currentApplet = null;
@@ -318,7 +318,7 @@ MouseArea {
                 Kirigami.Heading {
                     Layout.fillWidth: true
                     visible: panelSpacerWidth.visible
-                    text: i18nd("plasma_shell_org.kde.plasma.desktop", "Spacer width")
+                    text: i18n("Spacer width")
                     level: 3
                     horizontalAlignment: Text.AlignHCenter
                 }
-- 
GitLab

